modules/dxp/apps/antivirus/antivirus-async-store/src/main/java/com/liferay/antivirus/async/store/internal/scheduler/AntivirusAsyncFileStoreSchedulerJobConfiguration.java

```diff
@@ -265,6 +264,9 @@ private void _scheduleAntivirusScan(Path rootPath, Path filePath) {
 	@Reference
 	private MessageBus _messageBus;
 
+	@Reference(target = "(rootDir=*)")
+	private ServiceReference<Store> _storeServiceReference;
```

Comment:
Is it expected that there is only one `Store` service in use at any given time?

Reply:
At least with `rootDir=*`?

Reply:
Actually, I just added it back in the same form it had originally. We accidentally stopped using it due to refactoring in https://github.com/liferay/liferay-portal/commit/be240f78f32f215b7fe759edb567146224081ce2 and later the unused variable was removed by https://github.com/liferay/liferay-portal/commit/0203f6d64afc1ef55303a0675fe3316b7d8e5b09 . It looks like the NullPointerException started with that refactor.